packages/tabs/src/Tabs.tsx

```diff
@@ -60,6 +60,23 @@ const disabledStyle = css`
   cursor: not-allowed;
 `;
 
+function useDocumentActiveElement() {
+  const [activeEl, setActiveEl] = React.useState<Element | null>(null);
+
+  const handleFocusIn = useCallback(() => {
+    setActiveEl(document.activeElement);
+  }, [setActiveEl]);
```

Comment:
I don't think this needs `setActiveEl` as it should be stable between renders

Reply:
👍